### DISCUSSION TITLE
Calendar: Add support of new parameter "disableButton"

### DISCUSSION THREAD

**Discussion by dougmbarcellos** [primefaces/community#1665] [PrimeNG]
This new parameter should disable calendar button.

Situation: this new parameter is necessary since I'm not using disable property from FormControl, instead, we have a new custom property "readOnly". Therefore a new parameter in calendar component would be very useful for us.

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz example so we can investigate and identify the issue?
